crates/cli-support/Cargo.toml

```diff
@@ -28,3 +28,10 @@ wasm-bindgen-threads-xform = { path = '../threads-xform', version = '=0.2.92' }
 wasm-bindgen-wasm-conventions = { path = '../wasm-conventions', version = '=0.2.92' }
 wasm-bindgen-wasm-interpreter = { path = "../wasm-interpreter", version = '=0.2.92' }
 unicode-ident = "1.0.5"
+
+[dev-dependencies]
+insta = "1.39.0"
+
+[profile.dev.package]
+insta.opt-level = 3
+similar.opt-level = 3
```

Comment:
Can you leave a comment here saying why this is necessary?

Reply:
It's not necessary, but it's from the recommended setup: https://insta.rs/docs/quickstart/#optional-faster-runs

Rust compiles all build dependencies without any optimizations by default.

Reply:
That's a known fact I believe, this is only necessary when running your tests takes much longer than your compilation time. I think it this case its the reverse: compiling with optimizations just makes our compilation time longer with barely a benefit during runtime, because our test runtime should be around a second or so.

So I would remove this unless you think its not making the job actually run slower in CI?

Reply:
>  That's a known fact I believe

I didn't want to assume, Rust has a lot of lesser known corners! 😄